lib/client/api.go

```diff
@@ -703,7 +703,9 @@ func (tc *TeleportClient) Login() error {
 
 	// generate a new keypair. the public key will be signed via proxy if our password+HOTP
 	// are legit
-	priv, pub, err := native.New().GenerateKeyPair("")
+	keygen := native.New()
+	defer keygen.Close()
+	priv, pub, err := keygen.GenerateKeyPair("")
```

Comment:
for now I'm setting that variable to zero and it kind of makes these pools irrelevant (good), the only side effect was leaking goroutines but I fixed in throughout teleport code.